.github/workflows/release-test-version.yml

```diff
@@ -0,0 +1,132 @@
+name: 'Manual Release Test Version'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Specify a version for the test release (ex. test-browser)"
+        required: true
+        type: string
+        default: "test-version"
```

Comment:
IMO it's better to ask for a suffix that will then be appended to the current version. This way it will be clearer which version we're branching off of. Not a strong opinion though.

Reply:
We can probably debate it in the future although it doesn't feel mandatory to have the current version there in favor of a more expressive name 🤔

Reply:
One doesn't negate the other: `1.1.0-test` is more expressive and has more info that `test-version`

Reply:
I was thinking specifically as browser as an example, it would remove the possibility of having a full custom name like it was used recently if we always append to the end 🤔 
But it could be a good default (ex. if the version field is not specified, we just append `test` to the version)

Reply:
> it would remove the possibility of having a full custom name like it was used recently if we always append to the end

Do we need this?
In my opinion. having a version for the browser builds makes a lot of sense: the browser branch has been frequently synced with main, and having a version there makes it easier to distinguish different builds